app/kubemci/cmd/getstatus.go

```diff
@@ -77,7 +79,12 @@ func validateGetStatusArgs(options *GetStatusOptions, args []string) error {
 	}
 	// Verify that the required params are not missing.
 	if options.GCPProject == "" {
```

Comment:
You will also need to update the flag description above in addFlags to say this is optional in flag description.

Reply:
fixed.